src/app/oktaAuthCallback/oktaAuthCallback.spec.js

```diff
@@ -73,26 +73,26 @@ describe('oktaAuthCallback', function () {
 
 
   describe('onSignInFailure()', () => {
-    it('should log error message', () => {
+    it('should log error message when error is an Error instance', () => {
```

Comment:
I removed two tests because I consolidated the tests and gave them more descriptive names